### ISSUE TITLE
Dependency issues with the new Astyanax release - 2.0.0-SNAPSHOT

### ISSUE THREAD

**Issue by opuneet** [Netflix/astyanax#470]
I'm working with a customer (fortune 500, blah blah) that has done a bunch of Astyanax work for a POC, and are interested in targeting DSE 4.0 and Cassandra 2.0. For a few different reasons, I would like to see how reasonable it would be to get them to move to Astyanax 2.0 and the java driver. 

I got their POC code and started to try porting/building it to 2.0, but having non-obvious dependency issues.

The main issue appears to be that adding a pom dependency on astyanax-2.0.0-beta-SNAPSHOT or astyanax-all-2.0.0-beta-SNAPSHOT creates an implicit dependency on astyanax-core-2.0.0-beta-SNAPSHOT, but that jar file is not built (by checking out the 2.0-beta version from github and building). The javadoc and source jars are built:
astyanax-core/build/libs/astyanax-core-2.0.0-beta-SNAPSHOT-javadoc.jar
astyanax-core/build/libs/astyanax-core-2.0.0-beta-SNAPSHOT-sources.jar
but no main jar. 

Not sure if I'm doing something wrong, or if there actually is a broken dependency.

tl;dr how do I get a simple maven project to build against 2.0-beta-snapshot?
